src/clients/archwayd/index.js

```diff
@@ -46,11 +58,16 @@ class DefaultArchwayClient {
     return this.#keys;
   }
 
+  get query() {
+    return this.#query;
+  }
+
   get tx() {
     return this.#tx;
   }
 
   parseArgs(args = []) {
+    // TODO: add support for --home
```

Comment:
what add support for `--home` means? Should we also track this over an issue?

Reply:
I think it's not necessary since we will remove the dependency on `archwayd`